src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudIgnoreProxyIT.java

```diff
@@ -70,60 +70,80 @@ public class ComputeEngineCloudIgnoreProxyIT {
     private static ComputeClient client;
     private static Map<String, String> label = getLabel(ComputeEngineCloudIgnoreProxyIT.class);
     private static ComputeEngineCloud cloud;
-    private static FreeStyleProject project;
-    private static Node worker;
 
     @BeforeClass
     public static void init() throws Exception {
         log.info("init");
         initCredentials(jenkinsRule);
         cloud = initCloud(jenkinsRule);
         client = initClient(jenkinsRule, label, log);
-
-        InstanceConfiguration instanceConfiguration = instanceConfigurationBuilder()
-                .numExecutorsStr(NUM_EXECUTORS)
-                .labels(LABEL)
-                .template(NULL_TEMPLATE)
-                .googleLabels(label)
-                .oneShot(true)
-                .ignoreProxy(true)
-                .build();
-
         Jenkins jenkins = jenkinsRule.getInstance();
         jenkins.proxy = new ProxyConfiguration("127.0.0.1", 8080);
         jenkins.proxy.save();
         jenkins.save();
-        cloud.setConfigurations(ImmutableList.of(instanceConfiguration));
-        assertTrue(cloud.getInstanceConfigurationByDescription(instanceConfiguration.getDescription())
-                .isIgnoreProxy());
-
-        project = jenkinsRule.createFreeStyleProject();
-        Builder step = execute(Commands.ECHO, "works");
-        project.getBuildersList().add(step);
-        project.setAssignedLabel(new LabelAtom(LABEL));
-
-        FreeStyleBuild build = jenkinsRule.buildAndAssertSuccess(project);
-        worker = build.getBuiltOn();
-        assertNotNull(worker);
+        log.info("init completed");
     }
 
     @AfterClass
     public static void teardown() throws IOException {
         teardownResources(client, label, log);
     }
 
+    private static InstanceConfiguration getConfiguration(boolean ignoreProxy, String proxyLabel) {
+        return instanceConfigurationBuilder()
+                .description("IgnoreProxy=" + ignoreProxy)
+                .numExecutorsStr(NUM_EXECUTORS)
+                .labels(proxyLabel)
+                .template(NULL_TEMPLATE)
+                .googleLabels(label)
+                .oneShot(true)
+                .ignoreProxy(ignoreProxy)
+                .build();
+    }
+
+    private static FreeStyleProject createProject(String nodeLabel) throws IOException {
+        var project = jenkinsRule.createFreeStyleProject(nodeLabel);
+        Builder step = execute(Commands.ECHO, "works");
+        project.getBuildersList().add(step);
+        project.setAssignedLabel(new LabelAtom(nodeLabel));
+        return project;
+    }
+
     @Test
-    public void testWorkerIgnoringProxy() {
-        ComputeEngineInstance node = (ComputeEngineInstance) worker;
-        assertTrue(node.isIgnoreProxy());
+    public void testWorkerIgnoringProxy() throws Exception {
```

Comment:
previously one of the test was half completed in the `init()` while the other was fully in the test method.
Here keeping the test methods scoped correctly.